internal/runner/cypress.go

```diff
@@ -73,11 +74,12 @@ func (c Cypress) commandNameAndArgs(cmd string, testCases []string) (string, []s
 		return "", []string{}, err
 	}
 	idx := slices.Index(words, "{{testExamples}}")
+	specs := strings.Join(testCases, ",")
 	if idx < 0 {
 		words = append(words, "--spec")
-		words = append(words, testCases...)
+		words = append(words, specs)
 	} else {
-		words = slices.Replace(words, idx, idx+1, testCases...)
+		words = slices.Replace(words, idx, idx+1, specs)
```

Comment:
That's equivalent to:

```suggestion
		words[idx] = specs
```